### DISCUSSION TITLE
additional click required to refresh other sub window pane in verdi waveform viewer of synopsys

### DISCUSSION THREAD

**Discussion by js210711** [Xpra-org/xpra#4096] [Q&A]
when I clicked menu button for zoom in, zoom out, zoom out to full of verdi waveform viewer, waveform window pane is not promptly refreshed to reflect zoom action corresponding to zoom menu, respectively, when i click  additional click, zoom action is finally done. I wonder why this kind of symptom is shown considering that xpra is just forwading verdi interaction with mouse interface over network.

**Comment by totaam** [comment]
I don't know what verdi is, but if this is not an open source application, please try to reproduce the problem with one that is.

**Comment by totaam** [comment]
Not heard back, closing.
